scylla/tests/integration/retries.rs

```diff
@@ -232,3 +238,75 @@ async fn speculative_execution_panic_regression_test() {
         Err(err) => panic!("{}", err),
     }
 }
+
+/// This is a regression test.
+/// We check that speculative execution does not retry when the load balancing policy returns an empty plan.
+/// The test timeout is set to 5 seconds, so the speculative execution policy with a retry interval 6s cannot retry.
+#[tokio::test]
+#[ntest::timeout(5000)]
```

Comment:
:question: :thought_balloon: Can we be sure that 5 seconds is enough for running the whole test in the CI? What's the usual timeout of such session-building tests?